package.json

```diff
@@ -15,88 +15,65 @@
-  },
-  "devDependencies": {
-    "@babel/cli": "^7.10.5",
-    "@babel/core": "^7.10.5",
-    "@babel/plugin-proposal-class-properties": "^7.10.4",
-    "@babel/plugin-transform-runtime": "^7.10.5",
-    "@babel/preset-env": "^7.10.4",
-    "@babel/preset-react": "^7.10.4",
-    "@babel/register": "^7.10.5",
-    "@babel/runtime": "^7.10.5",
-    "@kadira/storybook": "^2.35.3",
-    "@rollup/plugin-alias": "^3.1.1",
-    "@rollup/plugin-babel": "^5.1.0",
-    "@rollup/plugin-commonjs": "^14.0.0",
-    "@rollup/plugin-node-resolve": "^8.4.0",
-    "@rollup/plugin-replace": "^2.3.3",
-    "@storybook/react": "^5.3.19",
-    "babelify": "^10.0.0",
-    "brfs": "^2.0.2",
-    "canvas": "^2.6.1",
-    "chai": "^4.2.0",
-    "chart.js": "2.6.0",
-    "cross-env": "^7.0.2",
-    "css-loader": "^3.6.0",
-    "debug": "^4.1.1",
-    "enzyme": "^3.11.0",
-    "enzyme-adapter-react-15": "^1.4.1",
-    "eslint": "^7.5.0",
-    "eslint-plugin-react": "^7.20.3",
-    "gh-pages": "^3.1.0",
-    "gulp": "^4.0.2",
-    "jsdom": "^16.3.0",
-    "mocha": "^8.0.1",
-    "rcolor": "^2.0.2",
-    "react": "^0.14 || ^15.0.0-rc || ^15.0",
-    "react-addons-test-utils": "^15.6.2",
-    "react-dom": "^0.14 || ^15.0.0-rc || ^15.0",
-    "react-test-renderer": "15",
-    "rimraf": "^3.0.2",
-    "rollup": "^2.22.1",
-    "rollup-plugin-generate-html-template": "^1.7.0",
-    "rollup-plugin-uglify": "^6.0.4",
-    "serve": "^11.3.2",
-    "sinon": "^9.0.2",
-    "webpack": "^4.43.0",
-    "webpack-dev-server": "^3.11.0"
-  },
-  "peerDependencies": {
-    "chart.js": "^2.3",
-    "react": "^0.14.0 || ^15.0.0 || ^16.0.0-beta || ^16.0.0",
-    "react-dom": "^0.14.0 || ^15.0.0 || ^16.0.0-beta || ^16.0.0"
-  },
-  "browserify-shim": {
-    "react": "global:React",
-    "react-dom": "global:ReactDOM",
-    "chart.js": "global:Chart"
-  },
-  "scripts": {
-    "clean": "rimraf lib es dist",
-    "build:cjs": "babel src --out-dir lib",
-    "build:es": "cross-env BABEL_ENV=jsnext babel src --out-dir es",
-    "build:umd": "cross-env BABEL_ENV=rollup NODE_ENV=development rollup -c -o dist/react-chartjs-2.js",
-    "build:umd:min": "cross-env BABEL_ENV=rollup NODE_ENV=production rollup -c -o dist/react-chartjs-2.min.js",
-    "build": "npm run clean && npm run build:cjs && npm run build:es && npm run build:umd && npm run build:umd:min",
-    "examples": "rollup -c example/rollup.config.js",
-    "examples:clean": "rimraf example/dist",
-    "examples:deploy": "npm run examples:clean && npm run examples && gh-pages -d example/dist",
-    "start": "npm run examples",
-    "lint": "eslint ./; true",
-    "test": "mocha test/config/setup.js test/__tests__/**/*",
-    "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
-  },
   "keywords": [
     "chart",
     "chart-js",
     "chart.js",
     "react-chartjs-2",
     "react chart.js"
   ],
-  "types": "./index.d.ts"
+  "scripts": {
+    "build": "microbundle-crl --no-compress --format modern,cjs",
```

Comment:
Missing script to publish the example to gh-pages

Reply:
Added